packages/react-provider/babel.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         extensions: [".css"],
       },
     ],
-    ["babel-plugin-react-remove-properties", { properties: ["data-test-id"] }],
+    ["babel-plugin-react-remove-properties", { properties: ["data-testid"] }],
```

Comment:
testid seemed to be the default so we don't need a file to tell it the new pattern